### DISCUSSION TITLE
Animation slow; able to disable it?

### DISCUSSION THREAD

**Discussion by hongtaoh** [lin-stephanie/astro-antfustyle-theme#5] [General]
I found that that animation between page loading is slow. Is this designed to be so or we can disable the animation?

**Comment by lin-stephanie** [comment]
@hongtaoh Thanks for bringing this up! Could you clarify which animation you mean? Is it the background animation while loading, or the sliding enter animation for content? A short video or more details would help understand better. Appreciate it!

**Reply by hongtaoh** [reply]
Sorry for the confusion. I mean the sliding enter animation for content. I tried https://antfu.me/, and it seems it comes from the original design. Is there a way to disable this? As I personally prefer loading the content fast. 


https://github.com/user-attachments/assets/a065e830-617e-4813-93af-aab0e17f9e6c

**Reply by lin-stephanie** [reply]
Got it! Yes, the sliding enter animation is part of the original design. If you'd like to disable it, the quickest way right now is to remove the styles under the `/* Slide Enter Animation */` comment in `src/styles/main.css`. 

Based on your feedback, I’ll add a configuration option in `src/config.ts` to toggle this in the next update! Thanks!

**Comment by lin-stephanie** [comment]
Version 1.3.0 now supports disabling the sliding enter animation by setting `slideEnterAnim: false` in `src/config.ts`.
